Sources/ComposableArchitecture/Internal/Create.swift

```diff
@@ -23,7 +23,7 @@
 import Combine
 import Darwin
 
-private class DemandBuffer<S: Subscriber> {
+class DemandBuffer<S: Subscriber> {
```

Comment:
This was only so I could use the existing `DemandBuffer` and would have to be changed back or marked as final